tools/vms-generator/utils/utils.go

```diff
@@ -559,8 +580,16 @@ func GetVMCirros() *v1.VirtualMachine {
 
 func GetTemplateFedora() *Template {
 	vm := getBaseVM("", map[string]string{"kubevirt-vm": "vm-${NAME}", "kubevirt.io/os": "fedora27"})
-	initFedora(&vm.Spec.Template.Spec)
-	addNoCloudDiskWitUserData(&vm.Spec.Template.Spec, "#cloud-config\npassword: fedora\nchpasswd: { expire: False }")
+	spec := &vm.Spec.Template.Spec
+	initFedora(spec)
+	addNoCloudDiskWitUserData(spec, "#cloud-config\npassword: fedora\nchpasswd: { expire: False }")
+
+	addINterface(spec, v1.InterfaceBindingMethod{
```

Comment:
is there ever a case where we add one but not the other? perhaps combine them into a single function (`setDefaultNetworking` or `setDefaultNetworkAndInterface` or perhaps `appendDefaultNetworkAndInterface`).